GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -132,6 +132,39 @@ private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string rep
             // Catch it in the calling function...
             var issues = await _gitHubClient.Issue.GetAllForRepository(owner, repository, issueRequest);
             issues = issues.Where(i => i.PullRequest == null).ToList();
+
+            // We now need to exclude the milestone
+            if (!String.IsNullOrEmpty(excludedMilestone) && !(excludedMilestone == "undefined"))
+            {
+                issues = issues.Where(i => i.Milestone == null || i.Milestone.Title != excludedMilestone).ToList();
+            }
+
+            // We now need to exclude all the issues that have labels that should be excluded
+            if (!String.IsNullOrEmpty(excludedLabels) && !(excludedLabels == "undefined"))
```

Comment:
Ditto about `string`, and `!=`

Reply:
fixed

---
In reply to: [247579159](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247579159) [](ancestors = 247579159)